docker/vue-app/src/form_editor/components/dialog_content/ConditionsEditorDialog.js

```diff
@@ -224,13 +242,23 @@ export default {
         getOperatorText(condition = {}) {
             const parFormat = condition.parentFormat.toLowerCase();
             let text = condition.selectedOp;
-            switch(text) {
+
+            const op = condition.selectedOp;

```

Comment:
This bit gets repeated a lot.  We might want to look at pulling it out into into a class/function that can be used by everything that needs it.